scripts/build_lib_ic.py

```diff
@@ -230,17 +231,20 @@ def main(libpath, verify=False):
 
 
 if __name__ == "__main__":
-    if len(sys.argv) == 2:
-        libpath = sys.argv[1]
-        main(libpath)
-    elif len(sys.argv) == 3 and sys.argv[2] == "--verify":
-        libpath = sys.argv[1]
-        if main(libpath, verify=True):
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
```

Comment:
What's the point of specifying `ArgumentDefaultsHelpFormatter` when none of your arguments have defaults?